src/ui/templates/insurance/account/sign-in/sign-in.njk

```diff
@@ -150,6 +150,4 @@
 
   </form>
 
-  <script src="/assets/js/mojFrontend.js" type="text/javascript"></script>
```

Comment:
Looks like MOJ frontend will be called in all pages now(?) If so, I think we should revert - currently only need to load the scripts for pages with password field (for show/hide functionality). Otherwise it seems the script could be run on every page when only ~1% of the pages need it